objectivec/GPBUtilities.m

```diff
@@ -1185,11 +1395,8 @@ void GPBSetMessageMapField(GPBMessage *self, GPBFieldDescriptor *field,
   if (dictionary) {
     GPBDataType keyDataType = field.mapKeyDataType;
     GPBDataType valueDataType = GPBGetFieldDataType(field);
-    NSString *keyStr = TypeToStr(keyDataType);
-    NSString *valueStr = TypeToStr(valueDataType);
-    if (keyDataType == GPBDataTypeString) {
```

Comment:
Isn't this still needed since the type is GPBString*Dictionary?

Reply:
Oops.. thank you. not sure how it got removed.